Modules/Sources/WordPressMedia/ImagePrefetcher.swift

```diff
@@ -0,0 +1,108 @@
+import UIKit
+import Collections
+
+@ImageDownloaderActor
+public final class ImagePrefetcher {
```

Comment:
Any particular reason of using a global actor, instead of making `ImagePrefetcher` an actor?

Reply:
I'm not sure if there are any situation where a new actor would be preferable to a global actor or another form of synchronization.

Reply:
Hmmm, I'm kind of on the opposite side of this ⬆️ . I would always define `actor` types, rather than define global actors. I see global actors as a way to synchronize multiple types, which I rarely need.

> It's also beneficial for performance as you can avoid unnecessary context switches. For example, when ImagePrefetcher calls _ = try? await downloader.image(for: request), there is no context switch as the code already runs on @ImageDownloaderActor.

Ah, I missed that `ImageDownloaderActor` is used by other types. Now it makes sense to me why `ImageDownloaderActor` global actor is used here.

Reply:
I agree, I think creating a new `actor` is a good starting point if there is no synchronization needed across different types/methods.